Add an alternative installation command for Windows

tolunaydundar opened the pull request:
Using "pip install thefuck" on Windows works, but the app itself does not.

Using "fuck" command on PowerShell/Terminal results with "ModuleNotFoundError: No module named 'imp'".

This alternative installation method fixes that error. See: https://github.com/nvbn/thefuck/issues/1434#issuecomment-2108618450

Thanks to @danthe1st

tolunaydundar committed:
Add an alternative installation command for Windows

Using "pip install thefuck" on Windows works, but the app itself does not.

Using "fuck" command on PowerShell/Terminal results with "ModuleNotFoundError: No module named 'imp'".

This alternative installation method fixes that error.
See: https://github.com/nvbn/thefuck/issues/1434#issuecomment-2108618450

Thanks to @danthe1st

danthe1st commented:
I think this is not specific to Windows but happens on other OSes as well - it's more that it depends on the Python version.

Also I think the better way to fix this would be for the maintainer to make a new release and publishing that to the Python package registry.

tolunaydundar committed:
Add PowerShell Set-ExecutionPolicy command to installation section

It is, as default, forbidden by PowerShell to run remotely signed apps to be run. This results with The Fuck failing to run.

In order to run The Fuck on PowerShell, the default ExecutionPolicy needs to be changed.
